server/application/application.proto

```diff
@@ -21,7 +21,7 @@ import "github.com/argoproj/argo-cd/v2/reposerver/repository/repository.proto";
 message ApplicationQuery {
 	// the application's name
 	optional string name = 1;
-	// forces application reconciliation if set to true
+	// forces application reconciliation if set to 'hard'
```

Comment:
```suggestion
	// forces application reconciliation if set to 'normal' or 'hard' - 'hard' ignores any repo cache
```

Maybe a bit more precise?

https://github.com/argoproj/argo-cd/blob/master/pkg/apis/application/v1alpha1/types.go#L289-L292

Reply:
If my reading of [the code](https://github.com/argoproj/argo-cd/blob/master/server/application/application.go#L665-L668) is correct, it defaults to "normal", and only switches if you send "hard". Do we want the text to reflect the functionality, or hide the implementation and describe the API instead?

Reply:
Good read, you're right. Let's stick with your wording.